### PR TITLE
DOMA-2686 added forced convertation of data fields to string for push notifications

### PR DESCRIPTION
**Pull request by akarjakin**

### CODE REVIEW

File: apps/condo/domains/notification/adapters/firebaseAdapter.js [hunk 1]
```diff
@@ -118,6 +119,32 @@ class FirebaseAdapter {
...
+    static prepareData (data = {}) {
+        const result = {}
+        const invalidFields = []
+
+        for (const key in data) {
+            if (data.hasOwnProperty(key)) {
+                result[key] = data[key]
+
+                if (!isString(data[key])) {
+                    result[key] = data[key].toString()
```
Comment (AleX83Xpert) [inline apps/condo/domains/notification/adapters/firebaseAdapter.js:135]: What if `data[key]` is `Number`?
image
```suggestion
                    result[key] = String(data[key])
```

File: apps/condo/domains/notification/adapters/firebaseAdapter.js [hunk 2]
```diff
@@ -118,6 +119,32 @@ class FirebaseAdapter {
...
+                    invalidFields.push([key, data[key]])
```
Comment (AleX83Xpert) [inline apps/condo/domains/notification/adapters/firebaseAdapter.js:136]: Why not Object? Object is more readable after stringifying than an array of arrays.
```suggestion
                    invalidFields[key] = data[key]
```

Comment (akarjakin) [reply apps/condo/domains/notification/adapters/firebaseAdapter.js:136]: invalidFields is an Array

Comment (AleX83Xpert) [reply apps/condo/domains/notification/adapters/firebaseAdapter.js:136]: I see, but why not Object instead?

**Review by AleX83Xpert — APPROVED** [review]

File: apps/condo/domains/notification/adapters/firebaseAdapter.js (hunk 2 above)
Comment (akarjakin) [reply apps/condo/domains/notification/adapters/firebaseAdapter.js:136]: actually no profit of having object here, also calculating object emptiness is more expensive then array emptiness :)

**Pull request merged**